domains/vaxhero.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "amitgkal"
+    },
+    "record": {
+        "A": "185.158.133.1"
```

```suggestion
        "A": ["185.158.133.1"]
```
A records must be in an array.